Gemfile

```diff
@@ -1,10 +1,11 @@
 source 'https://rubygems.org'
 
+ruby "2.1.10"
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.2.5.1'
-# Use sqlite3 as the database for Active Record
-gem 'sqlite3'
+gem "rails", "~> 4.2.6"
+# Use Postgres as the database for Active Record
+gem 'pg'
```

Comment:
It's just to keep heroku happy. I would have had to create the app with the option to exclude the ORM fully I think. I didn't bother with that, because I thought we might want it later as well, if we want to have an admin or something like that.